tests/cecil-tests/Makefile

```
@@ -0,0 +1,16 @@
+TOP=../..
+
+include $(TOP)/Make.config
+
+all-local::
+
+build:
+	msbuild
```

This is more quiet and also honors V=1 when doing `make V=1`:

```suggestion
	$(SYSTEM_MSBUILD)
```